Scripts/RemediationScripts/Remediate-EnableAADOnlyAuthenticationForSQLServers.ps1

```diff
@@ -310,64 +340,73 @@ function Enable-ADAdminForSqlServers
                 $adAdmin = Get-AzSqlServerActiveDirectoryAdministrator -ResourceGroupName $_.ResourceGroupName -ServerName $_.ServerName
             }
              
-            # Check if Azure AD admin are enabled on the SQL Server.
+            # Check if Azure AD Only Authentication are enabled on the SQL Server.
             if (($adAdmin|Measure-Object).Count -ne 0)
             {
-                $sqlServersWithADAdminEnabled += $sqlServerInstance
+                $sqlServerInstance.IsAADAdminPreviouslyConfigured = $true
+                $sqlServerInstance.EmailId = "NA(as AAD Admin is already configured)"
+                if ($adAdmin.IsAzureADOnlyAuthentication -eq $true)
+                {
+                    $sqlServersWithAADOnlyAuthEnabled += $sqlServerInstance
+                }
+                else
+                {
+                    $sqlServersWithAADOnlyAuthDisabled += $sqlServerInstance
+                }
             }
             else
             {
-                $SQLServersWithADAdminDisabled += $sqlServerInstance                                                 
+                $sqlServersWithAADOnlyAuthDisabled += $sqlServerInstance                                                 
             }
         }
         catch
         {
-            $SQLServersWithADAdminDisabled += $sqlServerInstance 
+            $sqlServersWithAADOnlyAuthDisabled += $sqlServerInstance 
```

Comment:
Just curious if any exception occurred, should we keep this in some skipped collection instead of disabled collection?

Reply:
Thanks for this Tarun, moving these into a separate skipped collection and printing it.